### PR TITLE
Fix Cursor plugin not updating info on image/channel blinking

### LINKED ISSUE #683: Cursor plugin does not update when blinking images
@dthatte reported that cursor info (coordinates and value) does not update when images are blinked in a channel (e.g., pressing up/down arrow keys to switch images in a channel).

### PR DESCRIPTION
**Pull request by ejeschke**
Fixes an issue where the Cursor plugin doesn't update its info when images are blinked in a channel, or the channels are blinked.  Fixes #683.

### CODE REVIEW

**Comment by ejeschke** [comment]
Ping @pllim, @dthatte.

File: ginga/ImageView.py [hunk 1]
```diff
@@ -1329,7 +1329,7 @@ def check_cursor_location(self):
         data_x, data_y = self.get_data_xy(self.last_win_x,
                                           self.last_win_y)
         if (data_x != self.last_data_x or
-                data_y != self.last_data_y):
```
Comment (pllim) [inline ginga/ImageView.py]: The original indent is the correct one.

**Comment by pllim** [comment]
@ejeschke , it seems like the error only appear with the following workflow:

* Open images in a channel.
* Start another channel and open/move an image there.
* Start `Blink` as a global plugin. Blink the channels.
* Start `Blink` as a local plugin in the first channel. Blink the images in the channel. Now I see the errors.

p.s. I am using a clean install (`ginga 2.7.2.dev1987`).

**Comment by ejeschke** [comment]
Excellent catch, @pllim!  I think that issue isn't related to this PR, but to the fact that this is a dual local/global plugin hybrid. I fixed it by updating the `redo()` method.  Can you check it again? Tnx

**Comment by pllim** [comment]
Seems to work now. Thanks!

**Comment by ejeschke** [comment]
tnx, @pllim. Merging.

**Pull request merged**